package.json

```diff
@@ -162,6 +162,7 @@
     "lint-staged": "^15.2.10",
     "minimist": "^1.2.8",
     "nock": "14.0.0-beta.7",
+    "node-fetch": "^3.3.2",
```

Comment:
why this library is being included ?

Reply:
This library is used to check the availability of exposed port before showing the result to the user. There is no reason to show the user result if exposed port is not reachable.

Reply:
fetch is part of Node.js so we don't need a 3rd party library